core/src/com/unciv/ui/map/TileGroupMap.kt

```diff
@@ -149,6 +163,6 @@ class TileGroupMap<T: TileGroup>(
     fun getMirrorTiles(): HashMap<TileInfo, Pair<T, T>> = mirrorTileGroups
 
     // For debugging purposes
-    override fun draw(batch: Batch?, parentAlpha: Float) { super.draw(batch, parentAlpha) }
-    override fun act(delta: Float) { super.act(delta) }
+//     override fun draw(batch: Batch?, parentAlpha: Float) { super.draw(batch, parentAlpha) }
+//     override fun act(delta: Float) { super.act(delta) }
```

Comment:
Why would you do this... this is uncommented be cause it helps debugging... it doesn't hurt performance at all

Reply:
True, I doubted the "doesn't hurt performance at all" but came to no conclusive evidence.